### PR TITLE
Graphql readonly

### PR DESCRIPTION
**Pull request by asood123**
This supports querying on Graphql backend for Events. Mutations coming separately.

### CODE REVIEW

File: migrations/20170116163901-create-Event-migration.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          autoIncrement: true
+        },
+
+        name: DataTypes.STRING,
+        description: DataTypes.TEXT,
+
+        createdById: {
```
Comment (xdamman) [inline migrations/20170116163901-create-Event-migration.js]: `createdByUserId`

File: server/models/Event.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+export default function(Sequelize, DataTypes) {
+
+  const Event = Sequelize.define('Event', {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+
+    createdById: {
```
Comment (xdamman) [inline server/models/Event.js]: createdByUserId

File: server/schema/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import {
```
Comment (xdamman) [inline server/schema/index.js]: move `/schema/` to `/graphql`

File: server/schema/query.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+import models from '../models';
+
+const queries = {
+  /*
+   * Given a group slug and an event slug, returns the event
+   */
+  getEvents: {
+    type: new GraphQLList(EventType),
+    args: {
+      slug: {
```
Comment (xdamman) [inline server/schema/query.js]: eventSlug ?

File: server/schema/types.js [hunk 5]
```diff
@@ -0,0 +1,390 @@
...
+          return event.name
+        }
+      },
+      description: {
+        type: GraphQLString,
+        resolve(event) {
+          return event.description
+        }
+      },
+      createdBy: {
```
Comment (xdamman) [inline server/schema/types.js]: createdByUser ?

File: test/schema.query.test.js [hunk 6]
```diff
@@ -0,0 +1,310 @@
...
+            UserId: user3.id 
+          })));
+
+        beforeEach(() => models.Response.create(
+          Object.assign(utils.data('response3'), { 
+            EventId: event1.id, 
+            TierId: tier2.id, 
+            GroupId: group1.id, 
+            UserId: user3.id 
+          })));
```
Comment (xdamman) [inline test/schema.query.test.js]: FYI, see what I've done with e.g. `User.createMany`

File: migrations/20170116163901-create-Event-migration.js (hunk 1 above)
Comment (asood123) [inline migrations/20170116163901-create-Event-migration.js]: done

File: server/models/Event.js (hunk 2 above)
Comment (asood123) [inline server/models/Event.js]: done

File: server/schema/index.js (hunk 3 above)
Comment (asood123) [inline server/schema/index.js]: done

File: server/schema/query.js (hunk 4 above)
Comment (asood123) [inline server/schema/query.js]: also updated

File: server/schema/types.js (hunk 5 above)
Comment (asood123) [inline server/schema/types.js]: also done

**Pull request merged**